cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/AbstractCluster.java

```diff
@@ -99,6 +100,7 @@ protected AbstractCluster(String namespace, String cluster, Labels labels) {
         this.cluster = cluster;
         this.namespace = namespace;
         this.labels = labels.withoutKind().withCluster(cluster);
+        this.volumeName = VOLUME_NAME;
```

Comment:
It seems strange to set it in constructor when there is no way to set the value. Why not do directly `protected String volumeName = VOLUME_NAME;`?

Reply:
If you want to stay with both variable and constant then I agree on doing what you are suggesting but at same time you proposed to get rid of one of them so maybe we could just do : 

`protected String volumeName = "data"`

wdyt ?

Reply:
Yeah, that sounds good. That should work.